Sources/TSCBasic/FileSystem.swift

```diff
@@ -1159,8 +1159,19 @@ public class RerootedFileSystemView: FileSystem {
     }
 }
 
+private var _localFileSystem: some FileSystem = LocalFileSystem()
```

Comment:
Temporarily introduced for `localFileSystem`'s explicit deprecated setter body to make sense.